Content.Server/Projectiles/EntityCollideAttemptEvent.cs

```diff
@@ -0,0 +1,9 @@
+public sealed class ProjectileCollideAttemptEvent : CancellableEntityEventArgs
+{
+    public readonly EntityUid Target;
+
+    public ProjectileCollideAttemptEvent(EntityUid target)
+    {
+        Target = target;
+    }
+}
```

Comment:
Isn't this just preventcollide event.

Reply:
i tried
```
[FATL] unhandled: System.InvalidOperationException: Duplicate Subscriptions for comp=Content.Shared.Projectiles.ProjectileComponent, event=PreventCollideEvent
```
shared projectile system is subscribed to this

Reply:
also if use prevent collide projectiles are reflected too early (about 2-3 character width distance)